spec/yoti/dynamic_share_service/extension/thirdparty_attribute_extension_spec.rb

```diff
@@ -10,11 +10,16 @@
     let :attr_name do
       'AttributeName'
     end
+    let :attr do
+      Yoti::DynamicSharingService::WantedAttribute.builder
+                                                  .with_name(attr_name)
+                                                  .build
```

Comment:
Is `WantedAttribute` what we should be using here for the third party attribute stuff?  I can't remember if this is what we use in the other SDK's for this, can you remember @echarrod or @davidgrayston ?

Reply:
You're right, this should be passed strings like this: `with_definitions('some-name', 'another-name')`